driverdataaggregation/main.py

```diff
@@ -167,9 +167,6 @@ def aggregation_scaling(self,
 
             # stop database after this set of permutations have finished
             exp.stop(db)
-            #Added to clean up db folder bc of issue with exp.stop()
-            time.sleep(5)
-            check_database_folder(result_path, logger)
```

Comment:
Why did you delete this code?

Reply:
so that checked the results folder for rbd files (rdb??) - we were having an issue with it not cleaning up the folders after a run, but the issue is solved, we also probably wanna know if this happens again so i deleted it